performance-tests/bench_2/test_controller/main.cpp

```diff
@@ -146,7 +198,17 @@ int ACE_TMAIN(int argc, ACE_TCHAR* argv[])
           << "                             incrementing numbers are assigned, but ID doesn't" << std::endl
           << "                             have to be a valid number, just a valid file name." << std::endl
           << "--overwrite-result           Write the result when using --result-id, even if" << std::endl
-          << "                             it would overwrite another result." << std::endl;
+          << "                             it would overwrite another result." << std::endl
+          << "--override-bench-partition-suffix SUFFIX     Override the system-wide partition suffix." << std::endl
+          << "--override-create-time N                     Override the system-wide creation time." << std::endl
+          << "--override-enable-time N                     Override the system-wide enabling time." << std::endl
+          << "--override-start-time N                      Override the system-wide starting time." << std::endl
+          << "--override-stop-time N                       Override the system-wide stopping time." << std::endl
+          << "--override-destruction-time N                Override the system-wide destruction time." << std::endl
+          << "--tags N [TAGS...]                           List of tags for which the user wants to collect" << std::endl
```

Comment:
I think the count as a parameter is a little clunky. How about just allowing "--tag <tag name>" to be specified multiple times? It's more verbose, but (I hope) less prone to usage errors.

Reply:
Reading through your PR, it dawns on me that what we'll probably really want is some way to specify tag "expressions" from the command line... something like "tagA && !(tagB || tagC)", but let's not worry about that for now. Tag names on their own are a good start.

Reply:
You could make use of yard to do something similar as what we're doing for queries.

Reply:
I agree with specifying the count being clunky. I think multiple `--tag` options would be better. Other alternatives that would also work would be something like [what I did for security debug logging tags which are comma separated](https://github.com/objectcomputing/OpenDDS/blob/94fb01271f49be83e38ce08d39ca6d42a4a5815b/dds/DCPS/debug.cpp#L53-L92) or requiring that `--tags` be the last option if it's passed and the rest of the positional arguments are treated as tags. Depending on how often tag are used since `test_controller` has a fixed number of positional arguments you could just treat the rest of the non-option arguments after the scenario as tags: `test_controller TEST_CONTEXT SCENARIO_ID [TAGS...]`.

Reply:
I'll change to use multiple "--tag" options for now.